docs/notes/previous_versions/v1.8.4.rst

```diff
@@ -0,0 +1,15 @@
+Version 1.8.4
+^^^^^^^^^^^^^
+
+This release adds the following **feature**:
+
+* XML profiles for `requester` and `replier`
+
+It also has the following **important bug fixes**:
+
+* Solved an issue when recreating a publishing participant with the same GUID (either on purpose or by chance)
+* Solved issue #1042, where a publisher could block on `write` for a long time when, after a large number of samples
```

Comment:
There could be a link to the issue

Reply:
Looking at the rest of the release notes, we have never referenced issues directly, so I am changing this to `an issue`